changelog/v1.18.3/memleak.yaml

```diff
@@ -0,0 +1,5 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/solo-projects/issues/7573
+    description: >-
+      Fix a memory leaking a log name.
```

Comment:
I see this is copied from the `main` PR but it doesn't seem very clear to me. Can we provide a better explanation?